boa_wasm/Cargo.toml

```diff
@@ -13,7 +13,7 @@ rust-version = "1.56"
 
 [dependencies]
 Boa = { path = "../boa", features = ["console"] }
-wasm-bindgen = "=0.2.78"
+wasm-bindgen = "=   0.2.78"
```

Comment:
What does this do?

Reply:
It was a typo, sorry, it has been fixed :)